examples/constant-force-optical-trap/extract-data.py

```diff
@@ -28,7 +28,7 @@
 
 original_data_directory = 'original-data' # location of original data (.xls files)
 processed_data_directory = 'processed-data' # location for where processed data is to be stored
-datasets = ['20R55_4T'] # prefixes of datasets to process (filenames are '%(prefix)_data.xls')
+datasets = ['20R55_4T'] # prefixes of datasets to process (filenames are ',format((prefix)_data.xls')
```

Comment:
`s/,/./` :)

Reply:
Yeah, I wasn't bothering with ones in comments :)